### PR TITLE
Allow tall layout arrange from right to left

### LINKED ISSUE #251: Anyway to make primary area of TallLayoutEngine on the Right side of the screen?
This isn't really an issue, but I couldn't find a discussion board or a forum.
(Might be a good idea to switch on GitHub's discussion feature for this repo).

### PR DESCRIPTION
**Pull request by cloudhan**
Fixes #251

### CODE REVIEW

File: src/workspacer.Shared/Layout/TallLayoutEngine.cs [hunk 1]
```diff
@@ -96,5 +98,10 @@ private int GetNumInPrimary()
         {
             return _numInPrimary + _numInPrimaryOffset;
         }
+
+        private int MaybeReverse(int x, int windowsWidth, int spaceWidth)
```
Comment (dalyIsaac) [inline src/workspacer.Shared/Layout/TallLayoutEngine.cs:102]: Could this be renamed to something like `CalcXPos`.

**Review by dalyIsaac — CHANGES_REQUESTED** [review]
LGTM, other than that one method name.

File: src/workspacer.Shared/Layout/TallLayoutEngine.cs (hunk 1 above)
Comment (cloudhan) [reply src/workspacer.Shared/Layout/TallLayoutEngine.cs]: I was struggling with that naming though, never though it can be that easy and clear.

**Review by dalyIsaac — APPROVED** [review]
LGTM!

**Comment by cloudhan** [comment]
@dalyIsaac Seem this PR needs another reviewer to proceeding? Can you ping someone for me.

**Comment by dalyIsaac** [comment]
Pinged the rest of the team, though may take a while.

**Comment by dalyIsaac** [comment]
In the mean time, you can actually copy the code for layout engine into your own config.

**Review by sitiom — APPROVED** [review]
Seems to work. LGTM!

**Review by system32uwu — APPROVED** [review]

**Pull request merged**